src/MovieDescription.js

```diff
@@ -0,0 +1,46 @@
+import React, {useEffect, useState} from "react";
+ import getMovieImg from './utils/getMovieImg'
```

Comment:
I think we may need to add a linter to this repo in the future.

Reply:
so smort!